### PR TITLE
feat: Add Instagram to API endpoint

### LINKED ISSUE #2654: Add Instagram to API endpoints
## User story

As a Trans Dim developer, I want to have access to Instagram data, as many groups use this as their primary web presence, in order that people can find and access these groups

## Acceptance criteria

- [x] All partners endpoint has instagram
- [x] One partner endpoint has instagram
- [x] Bruno examples are updated

## Implementation plan

_To be written by the developer_

<sub><a href="https://huly.app/guest/geeksforsocialchange?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzU1YWUwZDFlMDYzMmRkMDhlMjU4YTMiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6Incta2ltLWdlZWtzZm9yc29jaS02NzFlNzVmOS03ZTVlMTU1YzMwLTIxZjkwZCJ9.hyEqLf84Mo_RwSgrcgDf2XkS3ENZrWJc6e2JwN6IU0k">Huly&reg;: <b>PC-2661</b></a></sub>

### PR DESCRIPTION
**Pull request by kimadactyl**
Allows access to instagram field on the GraphQL endpoint

# Checklist:

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works,

Resolves #2654

## Description

- Adds IG methods to endpoint
- Updates Bruno collections
- Adds tests

### Motivation and Context

Instagram is one of the most crucial communications platforms for groups.

### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update

### How Has This Been Tested?

I have tested the endpoints in Bruno and added in an extra test.

<sub><a href="https://huly.app/guest/geeksforsocialchange?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzU1YjRjZTFlMDYzMmRkMDhlMjVhMDAiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6Incta2ltLWdlZWtzZm9yc29jaS02NzFlNzVmOS03ZTVlMTU1YzMwLTIxZjkwZCJ9.f6Ag-FfAEISYQTCX-MS2pj-hNqHas7YreI75xU1sJAQ">Huly&reg;: <b>PC-2662</b></a></sub>

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -179,7 +179,8 @@ GEM
     factory_bot_rails (6.2.0)
       factory_bot (~> 6.2.0)
       railties (>= 5.0.0)
-    ffi (1.15.5)
+    ffi (1.17.0)
+    ffi (1.17.0-x86_64-linux-gnu)
```
Comment (kimadactyl) [inline Gemfile.lock:183]: This was nessecary to get the bundle to resolve on my machine

File: app/models/partner.rb [hunk 2]
```diff
@@ -288,6 +288,10 @@ def twitter_url
     "https://twitter.com/#{twitter_handle}" if twitter_handle.present?
   end
 
+  def instagram_url
+    "https://instagram.com/#{instagram_handle}" if instagram_handle.present?
```
Comment (katjam) [inline app/models/partner.rb:292]: Couldn't immediately see where the `instagram_handle` is coming from - but appears it was already part of the model. Was it previously not being rendered as a link?

**Review by katjam — APPROVED** [review]
Doesn't seem like it can hurt anything!

**Comment by kimadactyl** [comment]
confirmed working on staging

**Pull request merged**